### PR TITLE
added directional light example

### PR DESCRIPTION
**Pull request by ieskudero**
based on the issue #15589. I have tried to create a simple example to show the use of a `THREE.DirectionalLight` and it's shadow, using the `THREE.HemisphereLight` as base.

### CODE REVIEW

**Comment by Mugen87** [comment]
Can you please rebase your branch to the latest version of `dev`? If you do this, it will be possible to check the new example with this link:

https://raw.githack.com/ieskudero/three.js/ieskudero-patch-1/examples/webgl_lights_directional.html

BTW: Please add the new example to [files.js](https://github.com/mrdoob/three.js/blob/dev/examples/files.js). Otherwise it's not selectable in the side navigation.

File: examples/webgl_lights_directional.html [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+
+				//SCENE
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color().setHSL( 0.6, 0, 1 );
+				scene.fog = new THREE.Fog( scene.background, 1, 5000 );
+
+				// LIGHTS
+
+				dirLight = new THREE.DirectionalLight( 0xffffff, 1 );
```
Comment (Mugen87) [inline examples/webgl_lights_directional.html]: I think you can omit the ctor parameters since you change the color in the next line.

**Comment by Mugen87** [comment]
Normally you solve the conflicts offline and then push the merge-commit. Maybe the easiest way is to start with a new branch from scratch. This would also mean you make a fresh PR.

**Comment by ieskudero** [comment]
> Normally you solve the conflicts offline and then push the merge-commit

I should have done so, yes, sorry about it. The conflict is in the `file.js`. The solution to this conflict is to accept both lines (`"webgl_lightningstrike"` and `"webgl_lights_directional"`) but I don't have write access.

I think it would be easier to solve it that way, but I can make another PR if you want, @Mugen87 .

And again, sorry for all the trouble.

**Pull request closed without merging**